`build` directory missing from published package for 2.0.0 causes runtime failure when `require` is used

`Error: Cannot find module '/Users/runner/work/markdownlint/markdownlint/node_modules/markdown-it-for-inline/build/index.cjs.js'`

<https://github.com/markdown-it/markdown-it-for-inline/blob/8e7eef13c79f3e7931059cf853285bb7e440b900/package.json#L17>

<https://www.npmjs.com/package/markdown-it-for-inline?activeTab=code>

<https://github.com/DavidAnson/markdownlint/actions/runs/7081100389/job/19269891516#step:5:1>

--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
   "module": "index.mjs",
   "exports": {
     ".": {
-      "require": "./build/index.cjs.js",
+      "require": "./dist/index.cjs.js",
       "import": "./index.mjs"
     },
     "./*": {

